eng/helix/content/runtests.cmd

```diff
@@ -10,7 +10,11 @@ set helixQueue=%5
 set arch=%6
 
 set DOTNET_HOME=%HELIX_CORRELATION_PAYLOAD%\sdk
-set DOTNET_ROOT=%DOTNET_HOME%\%arch%
+set DOTNET_ROOT=%DOTNET_HOME%
+if (%arch%==x86) (
+    set DOTNET_ROOT=%DOTNET_HOME%\%arch%
```

Comment:
Can this use `DOTNET_ROOT(x86)` too?

Reply:
Pinging some people on the dotnet team to get the formal definition of these environment variables. I probably can, but it'll cause a decent amount of churn as DOTNET_ROOT is used in a _lot_ of places.

Reply:
To be clear: These are comments and not requests for changes. Minimizing churn is an excellent goal.